### DISCUSSION TITLE
关闭自动播放下一个视频功能和播放列表定位

### DISCUSSION THREAD

**Discussion by twtan** [ywmoyue/biliuwp-lite#845] [Q&A]
### 问题描述

有播放列表的事情播放完会自动播放下一个视频，有时候不想自动播放下一关视频但是没法关闭
播放列表没法快速定位当前播放视频的位置，列表里面内容太多的话很难找到当前在播放的视频

### 描述解决方案

希望能增加关闭有播放列表自动播放下一个视频功能的开关，目前设置里面只有一个多P自动播放的关闭开关
切换到播放列表的时候能自动定位到当前播放视频的位置

### 代码贡献

否

**Comment by ywmoyue** [comment]
这两个功能在最新版本都有的

1. 播放设置中选择播完停止
2. 播放列表右下角有这个箭头按钮，目前可能会有点位置计算的错误

![image](https://github.com/user-attachments/assets/0298e6c2-9c87-4842-b1c1-935509c51303)


![image](https://github.com/user-attachments/assets/55bd6be0-2347-4a2e-9b48-42286f310808)
